l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -1227,7 +1252,8 @@ def invoiceCreate(self, fatt, fatturapa_attachment, FatturaBody, partner_id):
 
         invoice._onchange_invoice_line_wt_ids()
         invoice._recompute_dynamic_lines()
-        invoice.write(invoice._convert_to_write(invoice._cache))
+        write_vals = self._new_write_vals(invoice)
+        invoice.write(write_vals)
```

Comment:
Ma è ancora necessaria una `write` esplicita?
Non è sufficiente chiamare `_onchange_invoice_line_wt_ids`?
Come ad esempio https://github.com/OCA/account-invoicing/blob/d29f7ab523735525ea16886717a2b24affdcc9cc/stock_picking_invoicing/wizards/stock_invoice_onshipping.py#L552

Reply:
Era esattamente il mio dubbio. In effetti non mi sembrava necessario